utils/bin/lvim.ps1

```diff
@@ -0,0 +1,8 @@
+$env:HOME = "$HOME"
+$env:USER = "$HOME"
+$env:SHELL = "cmd"
+$env:TERMINAL = "$env:SHELL"
+$env:VIMRUNTIME = "$(scoop prefix neovim-nightly)/share/nvim/runtime";
+Set-Variable -name "LUNARVIM_RUNTIME_DIR" -Value "$HOME/.local/share/lunarvim"
+Set-Variable -name "LUNARVIM_CONFIG_DIR" -Value "$HOME/.config/lvim"
+nvim -u "$LUNARVIM_RUNTIME_DIR/lvim/init.lua" $args
```

Comment:
I suggest using some error handling since PowerShell allows it.

Reply:
That would be nice. But sadly, I know nothing about error handling in powershell